### PR TITLE
Pushing to github registry

### PR DESCRIPTION
**Pull request by radsoulbeard**

### CODE REVIEW

**Comment by srinikitha09** [comment]
This change is not relevant to our service right?

**Comment by radsoulbeard** [comment]
> This change is not relevant to our service right?

No, it is for completeness. In our service we need cf cli and node browser and that is already there.

**Comment by srinikitha09** [comment]
> This change is not relevant to our service right?

Sorry, it is .

**Review by srinikitha09 — APPROVED** [review]
👍🏼

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -16,5 +16,8 @@ jobs:
     - name: Run CI
       run: |
         echo ${{ secrets.DOCKERHUB_PASSWORD }} | docker login -u ${{ secrets.DOCKERHUB_USER }} --password-stdin
+        echo "${{ secrets.CR_PAT }}" | docker login https://ghcr.io -u ${{ secrets.CR_USER }} --password-stdin
```
Comment (srinikitha09) [inline .github/workflows/push.yml:19]: BTW is this variable correct? Shouldn't it be `CR_PASS`?

Comment (radsoulbeard) [reply .github/workflows/push.yml:19]: No it is CR_PAT for pass token

**Pull request merged**